libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -161,6 +161,14 @@ export interface AccountBridge<T extends TransactionCommon> {
     parentAccount?: Account | null | undefined;
     transaction?: T | null | undefined;
   }): Promise<BigNumber>;
+  applyReconciliation?: (
+    account: Account,
+    updatedRaw: AccountRaw,
+    next: Account
+  ) => boolean;
+  toAccountRaw?: (account: Account, accountRaw: AccountRaw) => void;
+  fromAccountRaw?: (accountRaw: AccountRaw, account: Account) => void;
+  mockAccount?: (account: Account) => void;
```

Comment:
this also need to be called "assign" as it's mutating account in place.

The idea is very interesting, however the name "mock" in it is not compatible with the idea of the bridge to me as it's specific to ONE kind of bridge: the mock one. the normal "js" implementation would have NOT to set it. **So I would recommend we could just rename it `initAccount` for now and only the mock bridge sets it?**

🤔  other thoughts => I'm not sure how we can converge to something unified, but it's relatively the same idea of "initializing an account with coin specifics" that we may also address via  https://ledgerhq.atlassian.net/browse/LIVE-2105 but it's indeed not exactly the same context (importAccountData was meant for the conversion from accountdata) **=> for now we could just have that `initAccount` you have defined that there actually a possibility we would use it for cross.ts itself! that may satisfies all our needs after all 👏**

Reply:
Fixed. Thanks for the feedback.